src/scan-imports.ts

```diff
@@ -3,6 +3,7 @@ import fs from 'fs';
 import glob from 'glob';
 import {parse} from '@babel/parser';
 import traverse from '@babel/traverse';
+import validate from 'validate-npm-package-name';
```

Comment:
I really don’t like adding dependencies to any project if I can help it, but here’s why I’d like to advocate for this rather than writing our own RegEx:

1. It’s `1 KB`
1. npm package names can actually be slightly-complicated, at least for a RegEx
1. this is managed by npm
1. this has a lot of tests

After checking out that repo, and discovering all the weird things npm packages could be named, I think this is better than any RegEx I could write!

Reply:
+1, makes sense to me (assuming that `1KB` also means 0 deps :)

Reply:
Oh you _know_ I’m all about that bundlephobia 🙂